src/index.js

```diff
@@ -4,6 +4,7 @@ import './locales'
 export { default as DataDimension } from './components/DataDimension/DataDimension'
 
 export { default as PeriodDimension } from './components/PeriodDimension/PeriodDimension'
+export { default as FixedPeriodSingleSelect } from './components/PeriodDimension/FixedPeriodSingleSelect'
```

Comment:
I think there's a need to rename the components internally (keeping the same names externally to not create breaking changes), since "PeriodDimension/PeriodDimension" and "PeriodDimension/FixedPeriodSingleSelect" might not be easy to understand. Previously, "PeriodDimension" (along with "DataDimension" etc) was a container for everything needed to select the respective dimension (i.e. as of now - a wrapper for the Transfer component). However now that PeriodDimension also houses another period selector, maybe they should be called something like "PeriodTransfer" and "FixedPeriodSelector" to separate the two?

Reply:
Also, calling it `FixedPeriodSingleSelect` when it actually consists of two single selects and a number input field might not be the most semantic as well 🙂